composer.json

```diff
@@ -24,7 +24,7 @@
     "psr/http-message": "~1.0"
   },
   "require-dev": {
-    "phpunit/PHPUnit": "~4.6",
+    "phpunit/phpunit": "^5.5",
```

Comment:
Change this constraint to `^4.6 || ^5.5` to allow this to test against PHP versions earlier than 5.5, please.